benchmarks/nel/test_project_nel_benchmark.py

```diff
@@ -0,0 +1,18 @@
+""" Testing all project steps. """
+
+from spacy.cli.project.run import project_run
+from spacy.cli.project.assets import project_assets
+from pathlib import Path
+
+
+def test_nel_benchmark_project():
```

Comment:
You can probably remove this file from this PR, as it's present in https://github.com/explosion/projects/pull/115 as well?

Reply:
Yes, same as above - should have been in https://github.com/explosion/projects/pull/115 only. I'll revert.